shop_richcatalog/cms_plugins.py

```diff
@@ -11,7 +12,16 @@ class CMSCatalogPlugin(CMSPluginBase):
     render_template = "catalog/catalog_grid_plugin.html"
 
     def render(self, context, instance, placeholder):
-        context['catalog_list'] = Catalog.objects.all()
+        context = super(CMSCatalogPlugin, self).render(
+            context, instance, placeholder)
```

Comment:
It doesn't look like this call to super was required before. I'm interested to know why we need it now.